inspirehep/records/marshmallow/literature/latex.py

```diff
@@ -30,9 +30,9 @@ def get_citations(self, data):
         citations = data.get("citation_count")
```

Comment:
if you define `citation_count` on the record class, you should be able to drop this method and just have
`citations = fields.Number(attribute='citation_count')` in the schema declaration.